packages/app/src/cli/models/app/loader.ts

```diff
@@ -352,18 +353,21 @@ class AppLoader<TConfig extends AppConfiguration, TModuleSpec extends ExtensionS
     return parseConfigurationFile(schema, filepath, this.abortOrReport.bind(this), decode)
   }
 
-  private showGlobalCLIWarningIfNeeded(nodeDependencies: {[key: string]: string}, packageManager: string) {
-    const hasLocalCLI = nodeDependencies['@shopify/cli'] !== undefined
-    // Show the warning IFF:
-    // - The current process is global
-    // - The project has a local CLI
+  private async showMultipleCLIWarningIfNeeded(directory: string) {
+    // Show the warning if:
+    // - There is a global installation
+    // - The project has a local CLI dependency
     // - The user didn't include the --json flag (to avoid showing the warning in scripts or CI/CD pipelines)
-    if (currentProcessIsGlobal() && hasLocalCLI && !sniffForJson() && !alreadyShownCLIWarning) {
+    // - The warning hasn't been shown yet during the current command execution
+    if (
+      (await isGlobalCLIInstalled()) &&
+      (await isLocalCLIInstalled(directory)) &&
+      !sniffForJson() &&
+      !alreadyShownCLIWarning
+    ) {
       const warningContent = {
-        headline: 'You are running a global installation of Shopify CLI',
-        body: [
-          `This project has Shopify CLI as a local dependency in package.json. If you prefer to use that version, run the command with your package manager (e.g. ${packageManager} run shopify).`,
-        ],
+        headline: 'There are two installations of the CLI: global and as a dependency of the project.',
+        body: [`We recommend to remove the @shopify/cli and @shopify/app dependencies from your package.json.`],
```

Comment:
And maybe link to this doc: https://shopify.dev/docs/apps/build/cli-for-apps#switch-to-a-global-executable-or-local-dependency 
?

Reply:
Sure thing! Can we use the headline to let the user know which one (and which version) will be used for the current command?
Eg

In the local case:
> **Using Shopify CLI from local dependency: v3.50.1**
> Shopify CLI was detected both as a global installation (v3.68.1) and as a local dependency (v3.50.1). We recommend removing the \`@shopify/cli\` and \`@shopify/app\` dependencies from your package.json.

In the global case:
> **Using global Shopify CLI: v3.68.1**
> Shopify CLI was detected both as a global installation (v3.68.1) and as a local dependency (v3.50.1). We recommend removing the \`@shopify/cli\` and \`@shopify/app\` dependencies from your package.json.

I think if someone is using one intentionally over the other, it's helpful to confirm which one is being used, and which version that is. Thoughts?

Reply:
Thank you! That makes total sense, and showing the versions will be very useful, because they are usually confused about what they are using.

About the recommendation, I think we prefer them to use the global CLI. And I guess most of the times they are using it by mistake. If they prefer to keep using that, they can check the documentation to uninstall the global one or simply ignore the warning.

One more doubt: is it ok to show this as an `info` banner? Or maybe better as a `warning`?